code/modules/mob/living/carbon/human/life/handle_breath.dm

```diff
@@ -9,12 +9,13 @@
 		return FALSE
 	return ..()
 
+
 /mob/living/carbon/human/handle_breath(list/air_info)
 	if(status_flags & GODMODE)
 		return
-	. = ..(air_info)
+	. = ..()
 	if(!.)
-		if(prob(10) && !(species?.species_flags & NO_BREATHE))
```

Comment:
change this back

Reply:
Why? All humans should have a species

Reply:
War against silent errors.
Change approved.